client/cli/go/cmds/topology.go

```diff
@@ -127,7 +127,8 @@ var topologyLoadCommand = &cobra.Command{
 		// Load current topolgy
 		heketiTopology, err := heketi.TopologyInfo()
 		if err != nil {
-			return err
+			fmt.Fprintf(stdout, "Unable to get topology information\n")
+			return fmt.Errorf("Unable to get topology information: %v", err)
                 }
```

Comment:
There is still an indentation error here.
Otherwise LGTM. Thanks, @humblec .

needs to be squashed with previous commit but keeping this commit msg.